### ISSUE TITLE
Dart DevTools display no Tabs (no content) after upgrade

### ISSUE THREAD

**Issue by JavierD8Belts** [flutter/devtools#3259]
Just after coming back to work from vacation I found a new **DevTools version**: **2.0.5**. Since then, when I try to open Dart DevTools on my project I see DevTools actually connected to my _running_ **Flutter web app**, but **the only output I get is the message "No tabs available for this application"**, no matter if I open DevTools within VSCode or directly on a browser (I have tried _Chrome_ and _Firefox_ by the time). If I try to open any tab in VSCode directly, i.e. the Widget Inspector, the message I get on screen is 'The "inspector" screen is not available for this application'.

On Chrome I see this message on Console: 
_Timed out trying to fetch flutter version from `ConnectedApp.initializeValues`._

This is the output of flutter doctor -v:
[✓] Flutter (Channel stable, 2.2.3, on macOS 11.5.1 20G80 darwin-x64, locale es-ES)
    • **Flutter version 2.2.3** at /Users/javierdelgadoesteban/development/tools/flutter
    • Framework revision f4abaa0735 (hace 6 semanas), 2021-07-01 12:46:11 -0700
    • Engine revision 241c87ad80
    • Dart version 2.13.4

[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.2)
    • Android SDK at /Users/javierdelgadoesteban/Library/Android/sdk
    • Platform android-30, build-tools 30.0.2
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6222593)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 12.5.1, Build version 12E507
    • CocoaPods version 1.10.0

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 4.1)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6222593)

[✓] VS Code (version 1.59.0)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.25.0

[✓] Connected device (1 available)
    • Chrome (web) • chrome • web-javascript • Google Chrome 92.0.4515.131

• No issues found!

So there's no problem to fetch flutter version as I see.

Flutter version: 2.2.3, channel stable
Dart version: 2.13.4
VSCode version: 1.59.0

**Comment by kenzieschmoll** [comment]
A few questions:
1) did you mean devtools 2.5.0, not 2.0.5?
2) how are you running your app?
3) when you say open DevTools directly in the browser, are you selecting this option from within VS code still?

**Comment by kenzieschmoll** [comment]
Just tried to reproduce. Ran flutter gallery as a web app via `flutter run -d chrome`. Opened the devtools link that was printed to console. DevTools opens connected to the flutter web app as expected. Perhaps this could be a VS Code only issue? cc @DanTup

**Comment by JavierD8Belts** [comment]
> A few questions:
> 
>     1. did you mean devtools 2.5.0, not 2.0.5?
> 
>     2. how are you running your app?
> 
>     3. when you say open DevTools directly in the browser, are you selecting this option from within VS code still?

1. Indeed, I mean DevTools 2.5.0, my mistake.
2. I'm running my app through VSCode.
3. Yes, I'm selecting the option form within VSCode.

Copying the uri provided on the console and pasting it on the DevTools url after "http://127.0.0.1:9100/#/?hide=debugger&ide=VSCode&uri=" didn't made a change.

**Comment by JavierD8Belts** [comment]
When I was gathering the data to post my problem I noticed the flutter version wasn't that one I expected. I'm using fvm to manage flutter versions, so I checked my current version: 2.2.2.
But down in VSCode status bar I read "Flutter: 2.0.1". I clicken on it, changed it to version 2.2.2, run the app again and 
Flutter DevTools worked again!
Hope it helps in case any other one faces the same kind of problem.
